docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+=====================================
+Welcome to the new Pootle 2.7.0 alpha
+=====================================
+
+*Not yet released*
+
+This is the first release of Pootle that combines the work of Evernote and
+Translate.
+
+Changes in Requirements
+=======================
+- Django >= 1.6.10 < 1.7
+- `Translate Toolkit <http://toolkit.translatehouse.org/download.html>`_ >=
+  1.12.0
+- Python >= 2.6
+- Redis - latest stable release.
+- Raven 5.0.0 or higher is now recommended for Sentry support on production
+  setups. While older versions should still work, they may encounter problems
+  in newer versions of Django or Python.
+
+
+Major Changes
+=============
+
+These are by no means exhaustive, check the `git log
+<https://github.com/translate/pootle/compare/stable%2F2.5.1...master>`_
+for more details.
+
+
+- Statistics calculations:
+
+  - Major rewrite of the way statistics are handled. RQ is used to manage the
+    calculation of invalidated statistics.  On the frontend side, stats are now
+    loaded asynchronously, thus any pending calculations no longer block page
+    loads.
+  - Custom word counting calculation method.
+
+
+- Big revamp of UI:
```

Comment:
Yes, a lot of stuff is still missing because it wasn't on our release notes or in evernote_changelog.